test/distributed/cases/dtype/binary.test

```diff
@@ -0,0 +1,16 @@
+-- @suite                                                                                                                                                                                
+-- @case
+-- @desc:test for binary datatype
+-- @label:bvt
+
+#Test for basic syntax.
+drop table if exists t1;
+create table t1(a binary(2));
+insert into t1 values("6");
```

Comment:
plz add binary type case about null,out of range ,insert chinese characters etc cases

Reply:
Test cases added.